docs/embedding_endpoints.md

```diff
@@ -1,26 +1,32 @@
 ---
-title: Configuring embedding backends 
-excerpt: Connecting MemGPT to various endpoint backends 
-category: 6580d34ee5e4d00068bf2a1d 
+title: Configuring embedding backends
+excerpt: Connecting MemGPT to various endpoint backends
+category: 6580d34ee5e4d00068bf2a1d
 ---
 
-MemGPT uses embedding models for retrieval search over archival memory. You can use embeddings provided by OpenAI, Azure, or any model on Hugging Face. 
+MemGPT uses embedding models for retrieval search over archival memory. You can use embeddings provided by OpenAI, Azure, or any model on Hugging Face.
 
 ## OpenAI
+
 To use OpenAI, make sure your `OPENAI_API_KEY` enviornment variable is set.
+
 ```sh
 export OPENAI_API_KEY=YOUR_API_KEY # on Linux/Mac
 ```
-Then, configure MemGPT and select `openai` as the embedding provider: 
-```
-> memgpt configure                                                                                    
+
+Then, configure MemGPT and select `openai` as the embedding provider:
+
+```text
```

Comment:
I think `text` makes sense here because shell will add weird syntax highlighting - however I noticed some of the prior outputs got converted from `text` to `shell` which may cause similar issues

Reply:
made it consistent with my latest commit. Believe I found them all anyway.